src/citrine/resources/table_config.py

```diff
@@ -88,6 +88,7 @@ class TableConfig(Resource["TableConfig"]):
         The query used to define the materials underpinning this table
     generation_algorithm: TableFromGemdQueryAlgorithm
         Which algorithm was used to generate the config based on the GemdQuery results
+
```

Comment:
My local `flake8` was complaining about this file, despite me not touching it. 🤷‍♂️